template/template.go

```diff
@@ -129,6 +131,16 @@ func (t *Template) Validate() error {
 		}
 	}
 
+	// Validate pushes
+	if t.Push != nil {
+		if t.Push.Address != "" {
+			log.Printf(`[WARN] The push configuration "address" has been renamed ` +
```

Comment:
There's no access to a UI object here, so I'm not sure the best way to warn this early in the init phase.